modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,29 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
+ *
+ * For streaming execution G-API has a special concurrent bounded queue to
+ * fetch next frame while current is being processed. This compilation argument
+ * specifies the capacity of this queue.
+ */
+struct queue_capacity
+{
+    size_t capacity;
```

Comment:
I believe default value would make sense here.

Reply:
Could you explain why it's needed ? From my point of view it's a bit confusing to pass `queue_capacity{}`

Reply:
I don't know how to add default value for `struct` in c++11

Reply:
`= 1`; ?

Reply:
Just self checked if am asking for something special, but seems am not

```c++
#include <iostream>
using namespace std;

struct queue_capacity
{
    size_t capacity = 1;
};

int main() {
	queue_capacity cap1{};
	queue_capacity cap2{2};
	std::cout << cap1.capacity << std::endl;
	std::cout << cap2.capacity << std::endl;
	return 0;
}
```

Reply:
Doesn't work :( Just added a `ctor`